CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Removed
 
 ### Fixed
+- Remove obsolete Backported classes for jackson 1.5 support
```

Comment:
This should be under `### Removed` and include the PR link at the end of the line, additionally fix casing of backported and Jackson:
```suggestion
- Removed obsolete backported classes for Jackson 1.5 support
```